playbooks/galaxy.yaml

```
@@ -24,6 +24,7 @@
           CONN_MAX_AGE: 0
           OPTIONS:
             sslmode: '{{ postgres_sslmode }}'
+            sslrootcert: /etc/pki/ca-trust/source/anchors/bundle-ca.crt
```

```suggestion
            sslrootcert: /etc/pki/tls/certs/ca-bundle.crt
```

https://github.com/ansible/awx-operator/blob/devel/roles/installer/defaults/main.yml#L443

I thought we needed `sslrootcert` only when the sslmode value was either `verify-full` or `verify-ca`, isn't the case ?